test/integration/prerender/test/index.test.js

```diff
@@ -358,33 +352,40 @@ const runTests = (dev = false) => {
     const html = await renderViaHTTP(appPort, '/catchall/another/value')
     const $ = cheerio.load(html)
 
-    if (dev) {
-      expect(
-        JSON.parse(
-          cheerio
-            .load(html)('#__NEXT_DATA__')
-            .text()
-        ).isFallback
-      ).toBe(true)
-    } else {
-      expect($('#catchall').text()).toMatch(/Hi.*?another\/value/)
-    }
+    expect(
+      JSON.parse(
+        cheerio
+          .load(html)('#__NEXT_DATA__')
+          .text()
+      ).isFallback
+    ).toBe(false)
+    expect($('#catchall').text()).toMatch(/Hi.*?another\/value/)
   })
 
   it('should support lazy catchall route', async () => {
-    const browser = await webdriver(appPort, '/catchall/third')
-    const text = await browser.elementByCss('#catchall').text()
-    expect(text).toMatch(/Hi.*?third/)
+    // Dev doesn't support fallback yet
+    if (dev) {
+      const html = await renderViaHTTP(appPort, '/catchall/notreturnedinpaths')
+      const $ = cheerio.load(html)
+      expect($('#catchall').text()).toMatch(/Hi.*?notreturnedinpaths/)
+    }
+    // Production will render fallback for a "lazy" route
+    else {
+      const browser = await webdriver(appPort, '/catchall/notreturnedinpaths')
+      const text = await browser.elementByCss('#catchall').text()
+      expect(text).toMatch(/Hi.*?notreturnedinpaths/)
+    }
   })
 
   if (dev) {
-    it('should show error when rewriting to dynamic SSG page', async () => {
-      const item = Math.round(Math.random() * 100)
-      const html = await renderViaHTTP(appPort, `/some-rewrite/${item}`)
-      expect(html).toContain(
-        `Rewrites don't support dynamic pages with getStaticProps yet. Using this will cause the page to fail to parse the params on the client for the fallback page`
-      )
-    })
+    // TODO: re-enable when this is supported in dev
+    // it('should show error when rewriting to dynamic SSG page', async () => {
```

Comment:
Leaving note here to un-comment once [this](https://github.com/zeit/next.js/pull/10510/files#r378596044) is addressed